app/views/OrderDetail/index.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import {
+    _cs,
+    isDefined,
+} from '@togglecorp/fujs';
+import {
+    Pager,
+    useInputState,
+    TextInput,
+    Header,
+    ListView,
+    ConfirmButton,
+    useAlert,
+} from '@the-deep/deep-ui';
+import {
+    IoSearchSharp,
+} from 'react-icons/io5';
+import {
+    gql,
+    useQuery,
+    useMutation,
+} from '@apollo/client';
+
+import {
+    OrderDetailsQuery,
+    OrderDetailsQueryVariables,
+    UpdateOrderStatusMutation,
+    UpdateOrderStatusMutationVariables,
+} from '#generated/types';
+
+import { orderDetail } from '#base/configs/lang';
+import useTranslation from '#base/hooks/useTranslation';
+import BookItem, { Props as BookItemProps } from '#components/BookItem';
+import OrderItem from '#components/OrderItem';
+
+import styles from './styles.css';
+
+const keySelector = (d: { id: string }) => d.id;
+// const labelSelector = (d: { name: string }) => d.name;
+
+const ORDER_DETIAL = gql`
+query OrderDetails($id: ID!, $bookPage: Int, $bookPageSize: Int) {
+    order(id: $id) {
+        id
+        orderCode
+        status
+        totalPrice
+        totalQuantity
+        bookOrders(page: $bookPage, pageSize: $bookPageSize) {
+            page
+            pageSize
+            totalCount
+            results {
+                id
+                edition
+                image {
+                    url
+                    name
+                }
+                isbn
+                price
+                quantity
+                title
+            }
+        }
+    }
+}
+`;
+
+const UPDATE_ORDER_STATUS = gql`
+mutation UpdateOrderStatus($id: ID!, $status: status) {
+    updateOrder(id: $id, data: {status: $status}) {
+        ok
+        errors
+        result {
+            id
+            status
+        }
+    }
+}
+`;
+
+const MAX_ITEMS_PER_PAGE = 10;
+interface Props {
+    className?: string;
+}
+function OrderDetail(props: Props) {
+    const { className } = props;
+
+    const [page, setPage] = React.useState<number>(1);
+    const [search, setSearch] = useInputState<string | undefined>(undefined);
+
+    const strings = useTranslation(orderDetail);
+    const alert = useAlert();
+    const routeParams = useParams<{
+        orderId: string;
+    }>();
+
+    const {
+        data: orderResponse,
+        // loading: orderLoading,
+    } = useQuery<OrderDetailsQuery, OrderDetailsQueryVariables>(
+        ORDER_DETIAL,
+        {
+            skip: !routeParams?.orderId,
+            variables: {
+                id: routeParams?.orderId ?? '',
+                bookPage: page,
+                bookPageSize: MAX_ITEMS_PER_PAGE,
+            },
```

Comment:
It's better to set variables as undefined when orderId is not defined and skip when there's no variable